src/react-navigation.js

```diff
@@ -1,5 +1,5 @@
 /*
- * @noflow - get/set properties not yet supported by flow. also `...require(x)` is broken #6560135
```

Comment:
Which version of flow does this break?

Alternative would be not to use `get` here.

Reply:
Basically we're relying on [this Flow functionality](https://github.com/facebook/flow/commit/6dea70976b4519a2e0fcc69977eed8699ea66243), which was already enabled in our `.flowconfig` with `unsafe.enable_getters_and_setters=true`. This is the same thing React Native does, so I think it's a safe bet.

Reply:
By the way, using this version of the PR (with typechecking enabled in the library definition) brings up a whole lot more (mostly) legitimate Flow errors on my personal project. I'm working through those errors right now to make sure nothing is wrong with the types in React Navigation.

Reply:
Greatness. Do you see this error https://github.com/react-community/react-navigation/pull/2616 ?

I guess this is due to an old version of Flow that a lot of users are still using (e.g. older React Native projects).

Reply:
I haven't seen that error in the process of building this PR. I'll comment with a thought on your PR.